### DISCUSSION TITLE
How to download JMA data?

### DISCUSSION THREAD

**Discussion by dzungpv** [open-meteo/open-meteo#421] [Q&A]
I could not found the guide to download JMA data, --help command not help much.

**Comment by patrick-zippenfenig** [comment]
Hi,

yes, you will need an JMA account or provide a server to JMA for them to upload data. However, it is unlikely that they are willing to grant access to non-academic institutions. I got access in collaboration with clients.

Best, Patrick

**Reply by dzungpv** [reply]
Thank you for clear reply.
